### DISCUSSION TITLE
Custom Event / Cell height

### DISCUSSION THREAD

**Discussion by mayhew09** [vkurko/calendar#297] [Q&A] [answered]
Is it possible to change the height of the cells? My calendar does not fill the screen even though the height is set to 100%, if I change i.e. the size of the cell in the css file to 28px it would fit my screen but then all events are all out of order as their position is obviously set programmatically.

**Comment by vkurko** [comment]
Please clarify what view you are talking about.

**Reply by vkurko** [reply, marked as answer]
If it is `timeGrid`, then there is [slotHeight](https://github.com/vkurko/calendar?tab=readme-ov-file#slotheight) option.

**Reply by mayhew09** [reply]
I'm using `timeGridWeek,timeGridDay`

**Reply by mayhew09** [reply]
Thanks, figured it out myself, it works with my view types too.
